instance/utils.py

```diff
@@ -92,6 +95,7 @@ def get_requests_retry(total=10, connect=10, read=10, redirect=10, backoff_facto
         redirect=redirect,
         backoff_factor=backoff_factor,
         status_forcelist=status_forcelist,
+        method_whitelist=method_whitelist,
```

Comment:
@shimulch Can you please add the note in PR description (deprecation of `method_whitelist` in favour of `allowed_methods` introduced in v`1.26.0`) here for documentation purposes?

Reply:
@arjunsinghy96 I've already added that in the `Notes` sections. :)